cylc/flow/scripts/lint.py

```diff
@@ -350,6 +356,39 @@ def check_for_deprecated_task_event_template_vars(
     return None
 
 
+BAD_SKIP_OUTS = re.compile(r'outputs\s*=\s*(.*)')
+
+
+def check_skip_mode_outputs(line: str) -> Dict:
+    """Ensure skip mode output setting doesn't include:
+
+    * succeeded _and_ failed: Mutually exclusive.
+    * submitted and started: These are emitted by skip mode anyway.
+
+    n.b.
+
+    This should be separable from ``[[outputs]]`` because it's a key
+    value pair not a section heading.
+    """
+
+    outputs = BAD_SKIP_OUTS.findall(line)
+    if outputs:
+        outputs = [i.strip() for i in outputs[0].split(',')]
+        if TASK_OUTPUT_FAILED in outputs and TASK_OUTPUT_SUCCEEDED in outputs:
+            return {
+                'description':
+                    'are mutually exclusive and cannot be used together',
+                'outputs': f'{TASK_OUTPUT_FAILED} and {TASK_OUTPUT_SUCCEEDED}'
+            }
+        pointless_outputs = get_unecessary_outputs(outputs)
+        if pointless_outputs:
+            return {
+                'description': 'are not required, they will be emitted anyway',
+                'outputs': f'{pointless_outputs}'
+            }
```

Comment:
I'm not sure this check is helpful. There is no harm in this configuration.

Reply:
Removed.